pkg/testutils/testcluster/testcluster.go

```diff
@@ -333,6 +333,9 @@ func NewTestCluster(
 		// concise and recognizable.
 		clusterArgs.ServerArgs.ClusterName = fmt.Sprintf("TestCluster-%s",
 			randutil.RandString(rng, 10, randutil.PrintableKeyAlphabet))
+	} else {
+		t.Logf("WARNING: TestCluster using non-unique ClusterName %q; "+
```

Comment:
👍

Reply:
Ignore that, I misread this for being the check at the message receiver's end. This place is better actually.